package.json

```diff
@@ -23,6 +23,8 @@
         "sdk:schemas:watch": "npm run --workspace packages/sdk build:schemas:watch",
         "copy-additional-files": "cpx \"src/**/*.{json,yml,ico}\" dist",
         "copy-additional-files:watch": "cpx \"src/**/*.{json,yml,ico}\" dist --watch",
+        "link-runtime": "bash .dev/scripts/linkRuntime.sh",
+        "unlink-runtime": "rm -rf node_modules/@nmshd && npm install",
```

Comment:
Please run

```bash
npx sort-package-json
```

Also, shouldn't these packages be called `(un)link-local-runtime`?

Reply:
`npx sort-package-json` had no effect, it only sorts the dependencies.

For the naming I would like to keep it as the only reason for linking is the local usage

Reply:
`npx sort-package-json` will not work as we have `npm-run-all` installed and that relies on script order

Reply:
sort-package-json typically also sorts scripts

Reply:
but yeah, for me that also doesn't work. I don't get you npm-run-all command, for me that is not the case but we can close this now.